cases/70lltu12/constant/boundaryData/BA/0/k

```diff
@@ -0,0 +1,61 @@
+58 
```

Comment:
I assume this is exp data from S. Kubacki, if not (e.g. output of some function) - please remove

Reply:
It is exp data.